services/notification/notifiers/mixins/message/sections.py

```diff
@@ -42,9 +42,11 @@ def get_section_class_from_layout_name(layout_name):
         return ImpactedEntrypointsSectionWriter
     if layout_name == "announcements":
         return AnnouncementSectionWriter
-    if layout_name == "header":
-        return HeaderSectionWriter
```

Comment:
Is this ever used considering these changes or has it become dead code that can be removed? 👀